src/components/WebexMeetingControlBar/WebexMeetingControlBar.jsx

```diff
@@ -151,9 +158,9 @@ export default function WebexMeetingControlBar({
     );
   };
 
-  const renderCollapseButton = (shown, onClick, ref) => (
+  const renderCollapseButton = (shown, onClick, ref, isVisible) => (
```

Comment:
instead of `isVisible`, you can just send it its tabIndex, which may be undefined